plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -16,35 +17,83 @@ class QueueType(Enum):
 class SlurmQueueConf:
     # Params are used to configure sbatch, for more info check:
     # https://github.com/facebookincubator/submitit/blob/master/submitit/slurm/slurm.py
-    nodes: int = 1
+
+    # maximum time for the job in minutes
+    time: int = 60
+    # number of cpus to use for each task
+    cpus_per_task: int = 10
+    # number of gpus to use on each node
     gpus_per_node: int = 1
+    # number of tasks to spawn on each node
     ntasks_per_node: int = 1
+    # number of nodes to use for the job
+    nodes: int = 1
+    # memory to reserve for the job on each node, in GB
```

Comment:
I think at this point we should just inline the memory.
the interpolation here is not helping.